dvpackager

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+while [ "${*}" != "" ] ; do
+    input="$1"
+    makelossless -j "${input}" && makeprores "${input}" && makeaccess "${input}" && makeframes "${input}" && makemetadata "${input}" && checksumpackage "${input}"
```

Comment:
wasn't it intended to remove makeprores in this case?

Reply:
Yep. Copied the wrong file.